ddtrace/api.py

```diff
@@ -25,7 +25,12 @@ def __init__(self, hostname, port, headers=None, encoder=None):
 
         # overwrite the Content-type with the one chosen in the Encoder
         self._headers = headers or {}
-        self._headers.update({'Content-Type': self._encoder.content_type})
+        self._headers.update({
+            'Content-Type': self._encoder.content_type,
+            'Meta-Lang': 'python',
+            'Meta-Lang-Version': PYTHON_VERSION,
+            'Meta-Lang-Interpreter': PYTHON_INTERPRETER,
+        })
```

Comment:
OK I think that approach is fine. Can you add a test to double check if those headers are properly sent when doing a call? You should find an example here: https://github.com/DataDog/dd-trace-py/blob/568134cc2cfd7956df10450efc3ef6c022e599c3/tests/test_integration.py#L223-L259

I'd say that these headers should be sent even for the `service` endpoint, so that we can still send tagged metrics even for services flushes.